installer/Migration/V5_0_0/lang-string/general.yaml

```diff
@@ -851,3 +848,9 @@ langStrings:
   -
     value: 'Delete Selected'
     unitId: delete_selected
+  -
+    value: 'Corporate Branding'
+    unitId: corporate_branding
+  -
```

Comment:
@Super-Chamajust  these two lines in general.yaml 855and 856

Reply:
OXD change related to this lang string will be merged with #1255 , Please rebase your PR from 5.0 after it is merged

Reply:
sure will rebase once yours in merged.

Reply:
Rebased